portal/config/eproms/Organization.json

```diff
@@ -2286,6 +2286,34 @@
       },
       "resourceType": "Organization"
     },
+    {
+      "extension": [
+        {
+          "timezone": "America/New_York",
+          "url": "http://hl7.org/fhir/StructureDefinition/user-timezone"
+        },
+        {
+          "research_protocols": [
+            {"name": "IRONMAN v3"}
```

Comment:
From the linked jira issue:
`Protocol Version: 4`
Beyond my love for magic numbers not lining up, doesn't 4 mean `IRONMAN v5` in this case?

Reply:
No... there were no changes necessary on our end between v3 and v4, so from our perspective they are equivalent.